src/feature/projects/components/ArticleFooter.tsx

```diff
@@ -40,20 +40,20 @@ const StyledTypography = styled(Typography)`
 
 export const ArticleFooter = (props: Props) => {
   const corsi = React.useMemo(() => {
-    return props.project.nodes[0].corsi[0];
+    return props.project.corsi[0];
   }, []);
 
   const bannerTitle = React.useMemo(() => {
     return `Inizia ora a studiare ${corsi.titolo}`;
   }, [corsi]);
 
   const nextProject = React.useMemo(() => {
-    return props.nextProject.nodes[0];
+    return props.nextProject;
   }, []);
 
   const nextProjectUrl = React.useMemo(() => {
-    const courseSlug = props.project.nodes[0].project_category[0].slug;
-    let slug = props.nextProject.nodes[0].titolo;
+    const courseSlug = props.project.project_category[0].slug;
+    let slug = props.nextProject.titolo;
     slug = slug
       .replace(/\s/g, "-")
       .replace(/[^a-zA-Z0-9-]/g, "")
```

Comment:
per fare questa filippinata ho creato una utils, apriti in issue per sostituirla così e cambiassimo la regex è tutto furbo